docs/t-sql/statements/delete-transact-sql.md

```diff
@@ -209,7 +209,7 @@ DELETE
  TOP cannot be used in a DELETE statement against partitioned views.  
   
 ## Locking Behavior  
- By default, a DELETE statement always acquires an intent exclusive (IX) lock on the table object it modifies, and holds that lock until the transaction completes. With an intent exclusive (IX) lock, no other transactions can modify data; read operations can take place only with the use of the NOLOCK hint or read uncommitted isolation level. You can specify table hints to override this default behavior for the duration of the DELETE statement by specifying another locking method, however, we recommend that hints be used only as a last resort by experienced developers and database administrators. For more information, see [Table Hints &#40;Transact-SQL&#41;](../../t-sql/queries/hints-transact-sql-table.md).  
+ By default, a DELETE statement acquires an intent exclusive (IX) lock on the table it modifies, and and a exclusive (X) Key, Page, or Row lock for each row deleted, and holds those locks until the transaction completes. With an intent exclusive (IX) lock, other transactions can read modify data, but can't take a an exclusive table lock. You can specify the TABLOCK table hint to override this default behavior and force DELETE to use an exclusive (X) table lock, however, we recommend that hints be used only as a last resort by experienced developers and database administrators. For more information, see [Table Hints &#40;Transact-SQL&#41;](../../t-sql/queries/hints-transact-sql-table.md).  

```

Comment:
```suggestion
 By default, a DELETE statement acquires an intent exclusive (IX) lock on the table it modifies, and an exclusive (X) Key, Page, or Row lock for each row deleted, and holds those locks until the transaction completes. With an intent exclusive (IX) lock, other transactions can read data but can't take an exclusive table lock. You can specify the TABLOCK table hint to override this default behavior and force DELETE to use an exclusive (X) table lock, however, we recommend that hints be used only as a last resort by experienced developers and database administrators. For more information, see [Table Hints &#40;Transact-SQL&#41;](../../t-sql/queries/hints-transact-sql-table.md).  

```

Reply:
@dbrownems Based on your resulting change:
1) "read modify" please clarify your intention. If it is the intention to change from "no other transactions can modify data" to "other transactions can read data" without specifying NOLOCK, can we be more specific as to the conditions that would allow other transactions to read data thru an IX?

Reply:
Should be:

By default, a DELETE statement acquires an intent exclusive (IX) lock on the table it modifies, and an exclusive (X) Key, Page, or Row lock for each row deleted, and holds those locks until the transaction completes. With an intent exclusive (IX) lock, other transactions can still read and modify data, but can't take an exclusive table lock.

>can we be more specific as to the conditions that would allow other transactions to read data thru an IX

We should just link the docs on Intent Locking ( https://docs.microsoft.com/en-us/sql/relational-databases/sql-server-transaction-locking-and-row-versioning-guide?view=sql-server-ver15#intent ) that explain that an IX lock doesn't block concurrent reads or writes, but only blocks an object-level X lock.